packages/swatch/src/SwatchGroup.ts

```diff
@@ -207,6 +215,11 @@ export class SwatchGroup extends SizedMixin(SpectrumElement, {
         const passThroughSwatchActions: ((swatch: Swatch) => void)[] = [];
         if (Object.keys(targetValues).length) {
             passThroughSwatchActions.push((swatch) => {
+                if (
+                    'selects' in targetValues &&
+                    targetValues['selects'] !== 'multiple'
+                )
+                    swatch.mixedValue = false;
```

Comment:
Altering state on element provided from a consumer could be surprising. Take a look at adding a Dev Mode message here ([like so](https://github.com/adobe/spectrum-web-components/blob/main/packages/progress-bar/src/ProgressBar.ts#L116-L136)) to prepare for the the case that a consumer _means_ to leverage `mixedValue`.

Reply:
@Westbrook Let me know if this looks good now. If the warning message needs to be customised do let me know.